test/e2e/helm_chart_test.go

```diff
@@ -0,0 +1,134 @@
+package e2e_test
+
+import (
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	helm "github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/cli/helm"
+	kube "github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/cli/kube"
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/cli/mongocli"
+
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/config"
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/model"
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/utils"
+)
+
+var _ = Describe("HELM charts", func() {
+	var userSpec model.UserInputs
+
+	var _ = BeforeEach(func() {
+		imageURL := os.Getenv("IMAGE_URL")
+		Expect(imageURL).ShouldNot(BeEmpty(), "SetUP IMAGE_URL")
+	})
+
+	var _ = AfterEach(func() {
+		By("Atfer each.", func() {
+			if CurrentGinkgoTestDescription().Failed {
+				GinkgoWriter.Write([]byte("Resources wasn't clean"))
+				utils.SaveToFile(
+					"output/operator-logs.txt",
+					kube.GetManagerLogs(config.DefaultOperatorNS),
+				)
+				SaveK8sResources(
+					[]string{"deploy"},
+					"default",
+				)
+				SaveK8sResources(
+					[]string{"atlasclusters", "atlasdatabaseusers", "atlasprojects"},
+					userSpec.Namespace,
+				)
+			}
+		})
+	})
+	It("[helm-ns] User can deploy operator namespaces by using HELM", func() {
+		By("User creates configuration for a new Project and Cluster", func() {
+			userSpec = model.NewUserInputs(
+				"only-key",
+				[]model.DBUser{
+					*model.NewDBUser("reader").
+						WithSecretRef("dbuser-secret-u1").
+						AddRole("readWrite", "Ships", "").
+						WithAuthDatabase("admin"),
+				},
+			)
+			userSpec.Clusters = append(userSpec.Clusters, model.LoadUserClusterConfig("data/atlascluster_basic_helm.yaml"))
+			userSpec.Clusters[0].Spec.Project.Name = userSpec.Project.GetK8sMetaName()
+			userSpec.Clusters[0].ObjectMeta.Name = "cluster-from-helm" // helm template has equal ObjectMeta.Name and Spec.Name
+			userSpec.Clusters[0].Spec.Name = "cluster-from-helm"
+		})
+		By("User use helm for deploying operator", func() {
+			helm.AddMongoDBRepo()
+			helm.InstallKubernetesOperatorNS(userSpec)
+		})
+		deployCluster(userSpec, 30006)
+	})
+
+	It("[helm-wide] User can deploy operator namespaces by using HELM", func() {
```

Comment:
what's the difference between this test case and the https://github.com/mongodb/mongodb-atlas-kubernetes/pull/204/files#diff-1c7d404bffb0725622f7419f40cd991c5765794f024aec084ab5d5fc5b3f6fafR45 ? I see they install the same helm charts and the same clusters?

Reply:
you're right , i'll change the call